README.md

```diff
@@ -1,2 +1,149 @@
-# Yoga-Class-Refactored
-A Refactored Yoga Application
+# About
+- Backend application demo using Spring Boot
+- Application to manage Yoga classes
+- Refactored of [old version](https://github.com/web-development-course-way/Yoga-Class)
+# Tech Stack
+- Docker
+- PostgreSQL & pgAdmin 
+- Java & Spring boot
+# Installation
+01. Clone Repository and open project in IDE
+02. [Set up database](https://github.com/web-development-course-way/Yoga-Class-Refactored/tree/enhancement/docs?tab=readme-ov-file#setting-up-database)
+03. Run application
+## Setting up database
+01. Run Docker
+02. Run Docker compose using following command `docker compose up` from the root of the project
```

Comment:
root? what is root folder? can you make it cd/yoga-class-refactor/ , then run docker compose up?